CommandLine/CommandLine.swift

```diff
@@ -79,7 +79,16 @@ public class CommandLine {
 
     return usedFlags
   }
-
+    
+  /**
+   * If true, when the stopper is encountered '--', we not only stop parsing options in general
+   * we also stop parsing multi-value options. This allows a user to indicate to an unbounded option
+   * to stop if it happens to be the last option in the option list, just before the non-option
+   * unparsed arguments.
+   *
+   */
+  public var stopperStopsMultiValueOptionParsing = false
```

Comment:
Well, you would not want it to if you wanted to maintain backwards compatibility.  The way it was before, the '--' would let you include option values that begin with '-'.  But in our case, especially with tags, and how we are using commands at the end, we really need a way to stop the tag option parsing.